integration-tests/spring-pojo-connection-limit-test/src/test/resources/microservice.yaml

```diff
@@ -0,0 +1,29 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+APPLICATION_ID: pojotest-it
+service_description:
+  name: pojo
+  version: 0.0.4
+servicecomb:
+  service:
+    registry:
+      address: http://127.0.0.1:30100
+  highway:
+    address: 0.0.0.0:7070
+    server:
+      connection-limit: 0
```

Comment:
consider dynamic update?

Reply:
I think it is not necessary yet

Reply:
discuss with customers, we should support this.

Reply:
Done

Reply:
already depend on pojo-test jar
and you want to reuse it's declare

so only need to append new config items, no need to declare everything again.